plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotWorkbenchActions.java

```diff
@@ -149,9 +155,23 @@ private static void openPreferencesDialogViaEvents(SWTBot bot) {
     bot.sleep(OPEN_PREFERENCES_DIALOG_DELAY_MS);
   }
 
-  /**
-   * Close the Welcome/Intro view, if found. Usually required on the first launch.
-   */
+  /** Open the view with the given ID. */
+  public static SWTBotView openViewById(SWTWorkbenchBot bot, String viewId) {
+    UIThreadRunnable.syncExec(
+        bot.getDisplay(),
+        () -> {
+          IWorkbenchWindow activeWindow = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
+          assertNotNull(activeWindow);
+          try {
+            activeWindow.getActivePage().showView(viewId);
+          } catch (PartInitException ex) {
+            logger.log(Level.SEVERE, "Unable to open view " + viewId, ex);
```

Comment:
Should the test fail in this case?

Reply:
It will as the `bot.viewById(viewId)` will fail below since the view won't be found.